app/src/main/java/io/swagslash/settlersofcatan/pieces/Edge.java

```diff
@@ -96,7 +96,7 @@ public boolean hasRoad() {
      * Determine if the player can build on this edge
      *
      * @param player the player to check for
-     * @return true iff the player has a road to an unoccupied vertex
```

Comment:
des hat schon gepasst, iff heißt if and only if 😂 aber is so auch ok 👍

Reply:
ok gut zu wissen xDD